src/components/Dashboard/TodoList.vue

```diff
@@ -85,10 +90,16 @@
 
                 <div v-if="isTypeCorrection(task) && isStatusDraft(task)" class="todo-subtitle">
                   <div>DRAFT</div>
-                  <v-btn x-small icon class="expand-btn">
-                    <v-icon>mdi-message-reply</v-icon>
+                  <v-btn
+                    v-if="task.comments.length > 0"
+                    class="expand-btn"
+                    outlined
+                    color="primary"
+                    :ripple=false
+                  >
+                    <v-icon small left style="padding-top: 2px">mdi-message-reply</v-icon>
```

Comment:
would a helper class work here? I'm not sure if class="pt-2" would work the same on a v-icon tho.. 🤔

Reply:
There isn't a Vuetify class that gives 2px padding/margin.

In this case it's very specific to this icon to push it down a bit for best display in this button so I think this inline style is acceptable.